recipes/earth_gebco.recipe

```diff
@@ -9,10 +9,10 @@
 #
 # Source: Information about master file, a title name (with underscores for spaces), planetary radius (km),
 #	name of z-variable, and z unit.
-# SRC_FILE=ftp://ftp.soest.hawaii.edu/pwessel/GEBCO/GEBCO_2023.nc
-# SRC_TITLE=GEBCO_Earth_Relief_2023
-# SRC_REF="GEBCO_Compilation_Group,_2023_GEBCO_2023_Grid"
-# SRC_DOI="https://doi.org/10.5285/f98b053b-0cbc-6c23-e053-6c86abc0af7b"
+# SRC_FILE=ftp://ftp.soest.hawaii.edu/pwessel/GEBCO/GEBCO_2024.nc
```

Comment:
This can't be right anymore

Reply:
Yes, I will revise the recipe when I finished processing it.

Reply:
I am not sure what to do with that. 

I understand that Paul put the grid there for a backup. Maybe we could use another place to put the data (like `/export/gmtserver/gmt`) or not save the data anywhere and use the original link (https://www.bodc.ac.uk/data/open_download/gebco/gebco_2024/zip/)

Reply:
We should use the original link if possible.

Reply:
I agree. The file from the original link is a zip. So, maybe this is an issue?  Should we: 

1. Use the original file (and at to the script a command to unzip it).
2. Save the unzip file in another directory in the server (as Paul did).

Reply:
Since the grid file is always on Paul's FTP, I think it's OK to use that link for this version. We can decide what to do in next version.

Reply:
Ok, but we should add the file to Paul's FTP but I don't know if we can. I think not.

Reply:
I thought it was already on Paul's FTP. If not, then I think we can't upload anything to it.

Can `SRC_FILE` be a local path?

Reply:
I usually get the original file, unzip it and move it to the staging directory. 

I think that the `SRC_FILE` only is needed is the file doesn't exist. I think this is the part of the script that do that:

```
# 5. Determine if this source is an URL and if we need to download it first
is_url=$(echo ${SRC_FILE} | grep -c :)
if [ $is_url ]; then	# Data source is an URL
	if [ ! -f ${SRC_BASENAME} ]; then # Must download first
		echo "srv_downsampler_grid.sh: Must download original source ${SRC_FILE}"
		curl -k ${SRC_FILE} --output ${SRC_BASENAME}
	fi
	SRC_ORIG=${SRC_FILE}
	SRC_FILE=${SRC_BASENAME}
fi
```

Reply:
Maybe we could use the zip file directly and unzip it as we do with the [emag2 recipe](https://github.com/GenericMappingTools/gmtserver-admin/blob/master/recipes/earth_mag.recipe). 

```
# SRC_FILE=https://www.ngdc.noaa.gov/geomag/data/EMAG2/EMAG2_V3_20170530/EMAG2_V3_20170530.zip
# SRC_PROCESS="rm -f EMAG2_V3_20170530.nc; unzip -n EMAG2_V3_20170530.zip"
```
I wonder if there is any good reason why Paul didn't do it and preferred to upload the data to the server.

Reply:
I think you can decide what the best way is.